src/styles/setings/active.scss

```diff
@@ -0,0 +1,14 @@
+.is-active {
+  margin: 0;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  bottom: 0;
+  background-color: #{$hover-color};
```

Comment:
The variable `$hover-color` is used here, but it is not defined in this file. Ensure that it is defined elsewhere in your SCSS files or imported correctly to avoid compilation errors.